test/opt/feature_manager_test.cpp

```diff
@@ -84,3 +84,51 @@ OpExtension "SPV_KHR_storage_buffer_storage_class"
   ASSERT_TRUE(context->get_feature_mgr()->HasExtension(
       libspirv::Extension::kSPV_KHR_storage_buffer_storage_class));
 }
+
+// Test capability checks.
+TEST_F(FeatureManagerTest, ExplicitlyPresent1) {
+  const std::string text = R"(
+OpCapability Shader
+OpMemoryModel Logical GLSL450
+  )";
+
+  std::unique_ptr<ir::IRContext> context =
+      BuildModule(SPV_ENV_UNIVERSAL_1_2, nullptr, text);
+  ASSERT_NE(context, nullptr);
+
+  ASSERT_TRUE(context->get_feature_mgr()->HasCapability(SpvCapabilityShader));
```

Comment:
Nit: These should be EXPECT_TRUE, EXPECT_FALSE.

Reply:
Done